src/addon/result-iterator.ts

```diff
@@ -33,7 +33,7 @@ export class ResultIterator<T> implements IterableIterator<T> {
   /**
    * Describe the result set schema.
    */
-  public describe(): string[][] {
+  public describe(): [string, string][] {
```

Comment:
if you could change this to `Array<[string, string]>` eslint will be happy :)

Reply:
done